### ISSUE TITLE
console.log(Element) output issue

### ISSUE THREAD

**Issue by X7md** [liriliri/luna#1]
# issue

when I try to log element there is some extra code log with element

```js
      const container = document.getElementById('container')
      const ele = document.createElement('div');
      const console = new LunaConsole(ele)
      console.log(container)
```
![image](https://user-images.githubusercontent.com/54203033/130808175-6e272e06-5817-480e-ac1d-87a269c09a26.png)

not sure but I try debugging and found this:

![image](https://user-images.githubusercontent.com/54203033/130807699-450dc7a6-c138-42a1-98fe-036ebb537f6b.png)
 
full code: 
```html
<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/luna-object-viewer/luna-object-viewer.css" />
    <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/luna-console/luna-console.css" />
</head>
<body>
    <div id="container">
    </div>

    <!--lib-->
    <script src="//cdn.jsdelivr.net/npm/luna-object-viewer/luna-object-viewer.js"></script>
    <script src="//cdn.jsdelivr.net/npm/luna-console/luna-console.js"></script>
    <!--lib-->

    <script>
      const container = document.getElementById('container')
      const ele = document.createElement('div');
      const console = new LunaConsole(container)
      console.log(container)
    </script>
</body>
</html>
```

**Comment by surunzi** [comment]
I forgot to delete some debugging code.

**Comment by erasec1994** [comment]
😁



---Original---
From: ***@***.***&gt;
Date: Tue, Aug 31, 2021 19:56 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [liriliri/luna] console.log(Element) output issue (#1)




 
I forgot to delete some debugging code.
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by X7md** [comment]
It's like console.log doesn't work with style element
```js
const container = document.getElementById('container')
      const ele = document.createElement('style');
      const console = new LunaConsole(container);
      console.log(ele);
```

**Comment by surunzi** [comment]
> It's like console.log doesn't work with style element
> 
> ```js
> const container = document.getElementById('container')
>       const ele = document.createElement('style');
>       const console = new LunaConsole(container);
>       console.log(ele);
> ```

Fixed in luna-console@0.2.1

**Comment by X7md** [comment]
Nice 💖
